src/content/configuration/dev-server.md

```diff
@@ -746,6 +746,8 @@ npx webpack serve --liveReload
 
 Notice that there's no way to disable it from CLI.
 
+W> Live reloading does not work with non-web targets like `node`, `async-node` etc.
```

Comment:
Sorry for approving earlier, but after thinking later I think naming all targets for which it works would be better. `etc` adds ambiguity for reader.

Reply:
Or maybe we can list just web targets which I believe is much easier to list as per https://webpack.js.org/configuration/target/#string

Reply:
yep 👍🏻